front/src/App.js

```diff
@@ -28,15 +31,47 @@ class App extends React.Component {
             )
     }
 
+    handleresearchIsVisible() {
+        this.setState({ researchIsVisible: !this.state.researchIsVisible })
+    }
+
+    handleSelectedTags(instrument) {
+        this.state.selectedTags.includes(instrument)
+            ? this.setState({
+                  selectedTags: this.state.selectedTags.filter(
+                      tag => tag !== instrument
+                  )
+              })
+            : this.setState({
+                  selectedTags: [...this.state.selectedTags, instrument]
+              })
```

Comment:
make it DRY ;)

(Don't Repeat Yourself)

Reply:
resolved applying setState to the ternary directly